src/components/ogre/OgreSetup.cpp

```diff
@@ -188,16 +188,37 @@ void OgreSetup::createOgreSystem() {
 	mPluginLoader.loadPlugin("Codec_FreeImage");
 	mPluginLoader.loadPlugin("Plugin_ParticleFX");
 	mPluginLoader.loadPlugin("RenderSystem_GL3Plus"); //We'll use OpenGL on Windows too, to make it easier to develop
+	mPluginLoader.loadPlugin("RenderSystem_Direct3D11");
+	mPluginLoader.loadPlugin("RenderSystem_Direct3D10");
+	mPluginLoader.loadPlugin("RenderSystem_Direct3D9");
 #endif
 
-	auto renderSystem = mRoot->getAvailableRenderers().front();
+	//auto renderSystem = mRoot->getAvailableRenderers().front();
+	const auto& renderers = mRoot->getAvailableRenderers();
```

Comment:
We used to support both DirectX and OpenGL, but in the end it just became a huge hassle since it meant I had to provide shaders for both GLSL and HLSL. So in the end we decided on only supporting OpenGL.
In the long run we want to move to a PBS based rendering flow, in which we hopefully won't need to write any shaders. And in the even longer run we want to see Vulkan support in Ogre, making away with both DirectX and OpenGL.

Reply:
Sorry, I didn’t come here for a long time. Yes, I wanted to make the game work on DirectX. But as I understand it, this is not worth doing. You can apply changes without this code.